CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - references to standards to test_configuration.toml
+- support for SystemErrorReport to test case glue:R0036_0
```

Comment:
well here it is still called R0036 instead of R0036_0

Reply:
That's right.
However, I don't want to keep things like that in my head until I finally merge everything together.
In the worst case, the merge is done fully automatically and it would be really hard to notice that this still needs to be updated in that case.

Reply:
fine for me.

Reply:
#17 is merged

Reply:
Yes, but it was reverted and reopened afterwards. Now we have to wait for #33 .